pinot-controller/src/main/resources/app/pages/TaskQueueTable.tsx

```diff
@@ -197,7 +202,9 @@ const TaskQueueTable = (props) => {
               headerTitle="Scheduling Errors"
               showSearchBox={false}
             >
-              
+              <Box p={3} style={{ wordBreak: 'break-all' }}>
```

Comment:
@satishwaghela please don't break the words here. Introduce horizontal scroll bars. Breaking the class path will make it unreadable.